profileService/internal/models/profile_test.go

```diff
@@ -0,0 +1,17 @@
+package models
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestProfileToJSON(t *testing.T) {
+	pro := new(Profile)
+	Id = "12345"
+	Email = "test@something.com"
+
+	expected := `{"id":"12345","username":"","email":"test@something.com","info":"","items":null,"LastUpdated":"0001-01-01T00:00:00Z"}`
```

Comment:
If the profile gets much bigger you might want to put this in a json file in `testdata` and load it during the test.

Reply:
ya that was the plan, so i could also test from json